orchestra/util.py

```diff
@@ -98,7 +98,11 @@ def is_installed(config, wanted_component, wanted_build=None, wanted_recursive_h
 def export_environment(variables: OrderedDict):
     env = ""
     for var, val in variables.items():
-        env += f'export {var}="{val}"\n'
+        if var.startswith("-"):
+            env += f'unset -v {var.lstrip("-")}\n'
```

Comment:
Maybe `var[1:]` is safer.
Shall we raise an error if `val` is not empty?

Reply:
Yeah good idea, I'll make this change

Reply:
Done